decred/tests/unit/dcr/test_txscript.py

```diff
@@ -812,6 +811,451 @@ def ssgenMsgTxWrongFirstOut():
     )
 
 
+class TestSubsidyCache:
+    def test_subsidy_cache_calcs(self, registerChain):
+        """
+        TestSubsidyCacheCalcs ensures the subsidy cache calculates the various
```

Comment:
~~TestSubsidyCacheCalcs ensures~~ -> Ensures

It might worth documenting that the test is based on `TestSubsidyCacheCalcs` from **dcrd**, though. 

Looks like all of the `TestSubsidyCache` methods need the same.